provider/pkg/gen/schema.go

```diff
@@ -624,6 +624,7 @@ func isDeprecated(description string) bool {
 	lowerDesc := strings.ToLower(description)
 	return strings.HasPrefix(lowerDesc, "deprecated") ||
 		strings.Contains(lowerDesc, ". deprecated") ||
+		strings.Contains(description, ". @Deprecated") ||
```

Comment:
This isn't strictly related but it resolved a conflict somewhere so I decided to add it.